js/packages/cli/src/commands/upload.ts

```diff
@@ -487,6 +487,15 @@ export function getAssetManifest(dirname: string, assetKey: string): Manifest {
   const manifest: Manifest = JSON.parse(
     fs.readFileSync(manifestPath).toString(),
   );
+  if (!('symbol' in manifest)) {
```

Comment:
Please allow undefined symbol and convert that to ""

Reply:
Updated PR